package.json

```diff
@@ -4,10 +4,8 @@
   "private": true,
   "license": "GPL-3.0",
   "scripts": {
-    "build-rust-ios": "cd rust/signer && make ios",
-    "ios": "yarn run build-rust-ios && yarn run cli run-ios",
-    "build-rust-android": "cd rust/signer && make android",
-    "android": "yarn run build-rust-android && yarn run cli run-android",
+    "ios": "npm run build-rust-ios && react-native run-ios",
```

Comment:
This is refering to `build-rust-ios`, but `build-rust-ios` is removed :see_no_evil:.